### DISCUSSION TITLE
Compiled with problems - version @11.1.1

### DISCUSSION THREAD

**Discussion by kristajg** [twilio-labs/paste#2258] [Office hours] [answered]
Hello, I just created a new project with paste as follows:

```
npm create react-app --template @twilio-paste my-supercool-app
cd my-supercool-app
npm start
```
It's compiling with problems that don't look serious but it happens on first load / refresh:
Compiled with problems:X

ERROR in ./node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1945:10-17

Should not import the named export 'props' (imported as 'S') from default-exporting module (only default export is available soon)

ERROR in ./node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1951:10-17

Should not import the named export 'props' (imported as 'S') from default-exporting module (only default export is available soon)
@shleewhite

**Comment by shleewhite** [comment]
Thanks for catching this bug!! This is an issue with our esbuild configuration. We are working on a fix, will update the thread when it is merged.

**Reply by shleewhite** [reply]
@kristajg It looks like this fix may take some time, so until it is ready you can roll `@twilio-paste/core` back to `10.20.0`.

**Comment by SiTaggart** [comment, marked as answer]
Hi @kristajg 

This is now fixed in `@twilio-paste/core@11.1.2`. Thanks for contributing to Paste and making it better!

**Reply by kristajg** [reply]
@SiTaggart Just upgraded and it's all clear. Works great, thank you!
